tutorials/how-to-setup-applesilicon-server-with-terraform-ansible/index.mdx

```diff
@@ -0,0 +1,229 @@
+
+```terraform
+#resources.tf
+resource "scaleway_apple_silicon_server" "server" {
+    name = "MyAwesomeServer"
+    type = "M2-M"
+    zone = "fr-par-1"
+}
+```
+
+6. Apply the Configuration: To apply this configuration, run the following commands in your terminal:
+
+```shell
+#Initialize Terraform
+terraform init
+#Plan the deployment
+terraform plan
+#Create the server
+terraform apply
+```
+
+When prompted, type yes to confirm the creation of the resources.
+
+7. Enable Virtual Private Cloud (VPC) and Private Network: To enhance the network setup, you can update the configuration to enable VPC option and attach a private network to your Apple Silicon server. Update your resources.tf file with the following
+
+```terraform
+#resources.tf
+resource "scaleway_vpc" "vpc01" {
+    name = "MyAwesomeVPC"
+}
+
+resource "scaleway_vpc_private_network" "pn01" {
+    name = "MyAwesomePN"
+    vpc_id = scaleway_vpc.vpc01.id
+}
+
+resource "scaleway_apple_silicon_server" "server" {
+    name = "MyAwesomeServer"
+    type = "M2-M"
+    zone = "fr-par-1"
+    enable_vpc = true
+    private_network {
+          id = scaleway_vpc_private_network.pn01.id
+    }
+}
+```
+8. Apply the Configuration Update: Finally, run the following command to apply the changes and update the server configuration
+
+```shell
+terraform apply
+```
+
+This will apply the new settings, ensuring that the server is launched within the specified VPC and connected to the private network.
+
+### Using Ansible
+
+1. Install Ansible:  If you don't have Ansible installed on your system, you can easily install it using pip, the Python package manager. Run the following command:
+
+```shell
+pip install ansible
+```
+
+2. Install Scaleway Ansible Collection: Scaleway provides an official Ansible collection that enables you to interact with Scaleway resources, such as creating servers. Install this collection using the ansible-galaxy command:
+
+```shell
+ansible-galaxy collection install scaleway.scaleway
+```
+
+3. Create a Directory: Now that you have Ansible and the Scaleway collection installed, create a directory for your Ansible project. This will help you keep everything organized.
+
+Run the following commands to create and navigate into the project directory:
+
+```shell
+mkdir apple_silicon_server_ansible
+cd apple_silicon_server_ansible
+```
+
+4. Create a Playbook: In Ansible, a playbook is a YAML file that defines the tasks to automate. Create a new file for your playbook, named create_applesilicon_server.yml:
+
+```shell
+touch create_applesilicon_server.yml
+```
+
+5. Define the Playbook Content: Open the create_applesilicon_server.yml file in your preferred text editor. Add the following content to define the task of provisioning an Apple Silicon server:
+
+```ansible
+---
+- name: Create Apple Silicon Server on Scaleway
+  hosts: localhost
+  gather_facts: no
+  tasks:
+    - name: Create an Apple Silicon server
+      scaleway.scaleway.scaleway_applesilicon_server:
```

Comment:
It is completely redundant with the terraform part of the tutorial. I fail to see what this steps provides. One useful alternative could be to either:
- split this tutorial in two (one part for terraform, the other for ansible)
- replace the ansible part with a part that show how to interconnect terraform and ansible using `local-exec` https://developer.hashicorp.com/terraform/language/resources/provisioners/local-exec